tests/run/coverage_installed_pkg.srctree

```diff
@@ -0,0 +1,71 @@
+# mode: run
+# tag: coverage,trace
+
+"""
+PYTHON setup.py build_ext -i
+PYTHON -c "import shutil; shutil.move('ext_src/ext_pkg', 'ext_pkg')"
+PYTHON -m coverage run coverage_test.py
+PYTHON -m coverage report
+"""
+
+######## setup.py ########
+from distutils.core import setup, Extension
+from Cython.Build import cythonize
+
+setup(ext_modules = cythonize([
+    'pkg/*.py*',
```

Comment:
Is it actually necessary for this test to compile the `.py` files, in addition to the `.pyx` files?

I'm asking for two reasons: it complicates the test setup (thus increasing the risk of potentially unrelated failures), and there is currently a bug in your branch that I fixed in latest master regarding the compilation of `__init__.py` files under Windows, so I'm getting [failures](https://ci.appveyor.com/project/cython/cython/builds/21403742) in appveyor. Merging master should resolve that, but if we can avoid this complication entirely, then that's clearly preferable.

Reply:
Done, changed to ``*.pyx``.